src/app/myworkflows/myworkflows.component.ts

```diff
@@ -1,15 +1,39 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+
+import {CommunicatorService} from '../shared/communicator.service';
+import {DockstoreService} from '../shared/dockstore.service';
+import {ProviderService} from '../shared/provider.service';
+import {WorkflowObservableService} from '../shared/workflow-observable.service';
+
+import {MyWorkflowsService} from './myworkflows.service';
+import {UserService} from '../loginComponents/user.service';
+
+
 
 @Component({
   selector: 'app-myworkflows',
   templateUrl: './myworkflows.component.html',
-  styleUrls: ['./myworkflows.component.css']
+  styleUrls: ['./myworkflows.component.css'],
+  providers: [MyWorkflowsService, ProviderService,
+              DockstoreService, CommunicatorService, WorkflowObservableService]
 })
-export class MyWorkflowsComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit() {
+export class MyWorkflowsComponent {
+  orgWorkflows = [];
+  oneAtATime = true;
+  constructor(private myworkflowService: MyWorkflowsService,
+              private userService: UserService,
+              private communicatorService: CommunicatorService,
+              private workflowobjService: WorkflowObservableService) {
+    userService.getUser().subscribe(user => {
```

Comment:
move things out of constructor and into ngOnInit, since the constructor is not really a part of the Angular life cycle.